modules/loadbalancer/variables.tf

```diff
@@ -69,7 +69,7 @@ variable "frontend_ips" {
   - `create_public_ip`          - (`bool`, optional, defaults to `false`) when set to `true` a new public IP will be
                                   created, otherwise an existing resource will be used;
                                   in both cases the name of the resource is controled by `public_ip_name` property
-  - `public_ip_resource_group`  - (`string`, optional, defaults to the Load Balancer's RG) name of a Resource Group
+  - `public_ip_resource_group_name`  - (`string`, optional, defaults to the Load Balancer's RG) name of a Resource Group
```

Comment:
Shall we always keep the same empty space between key - description for all properties or is it different on purpose here?

Reply:
this is just a mistake, I was fixing the `_name` suffix with `sed` or something.